### PR TITLE
[StyleCop] Fix warnings in BotBuilderRecognizerSamples

### PR DESCRIPTION
**Pull request by Bill7zz**
- Rename variables with Hungarian notation
- Add space after closing brace
- Document the return element in the method
- Reorder using directives
- Remove trailing whitespace

### CODE REVIEW

File: .NET/Samples/BotBuilder/Dialogs/RootDialog.cs [hunk 1]
```diff
@@ -17,6 +17,13 @@ public async Task StartAsync(IDialogContext context)
             context.Wait(this.MessageReceivedAsync);
         }
 
+        private static string GetCurrentCultureCode()
+        {
+            // Use English as default culture since the this sample bot that does not include any localization resources
```
Comment (VictorGrycuk) [inline .NET/Samples/BotBuilder/Dialogs/RootDialog.cs]: I think there is an extra `the` in the line `[...] since the this sample bot [...]`.

**Review by VictorGrycuk — APPROVED** [review]
LGTM!

**Comment by JuanAr** [comment]
@Bill7zz please fix the commit history: "Revert..."

**Review by JuanAr — CHANGES_REQUESTED** [review]
Feedback regarding commits

**Review by ceciliaavila — APPROVED** [review]
LGTM

File: .NET/Samples/BotBuilder/Dialogs/RootDialog.cs (hunk 1 above)
Comment (Aliandi) [reply .NET/Samples/BotBuilder/Dialogs/RootDialog.cs]: @Bill7zz this is still here!

File: .NET/Samples/BotBuilder/BotBuilderRecognizerBot.csproj [hunk 2]
```diff
@@ -125,8 +125,8 @@
     <Compile Include="Dialogs\QuantityPrompt.cs" />
     <Compile Include="Dialogs\RootDialog.cs" />
     <Compile Include="Controllers\MessagesController.cs" />
-    <Compile Include="Global.asax.cs">
-      <DependentUpon>Global.asax</DependentUpon>
+    <Compile Include="WebApiApplication.asax.cs">
+      <DependentUpon>WebApiApplication.asax</DependentUpon>
```
Comment (Aliandi) [inline .NET/Samples/BotBuilder/BotBuilderRecognizerBot.csproj:129]: Ln 129, 128 and 117 arw your changes or the changes that came from master?

**Review by Aliandi — APPROVED** [review]

**Review by JuanAr — APPROVED** [review]
LGTM

**Pull request closed without merging**